src/Umbraco.Web/Editors/PasswordChanger.cs

```diff
@@ -84,6 +85,11 @@ public PasswordChanger(ILogger logger, IUserService userService, HttpContextBase
                     return Attempt.Fail(new PasswordChangedModel { ChangeError = new ValidationResult("The current user is not authorized", new[] { "resetPassword" }) });
                 }
 
+                if (!currentUser.IsAdmin() && savingUser.IsAdmin())
+                {
+                    return Attempt.Fail(new PasswordChangedModel { ChangeError = new ValidationResult("The current user can't change password for specific user", new[] { "resetPassword" }) });
```

Comment:
Wording maybe should be "The current user cannot change password for the specified user" ?